homework_03/14_martin_damyanov/homework.py

```diff
@@ -0,0 +1,74 @@
+from enum import Enum
+import math
+
+class RelativePosition(Enum):
+    NO_COMMON_POINTS = 1
+    TOUCHING = 2
+    INTERSECTING = 3
+    SAME = 4
+
+def distance(a, b):
+    try:
+        assert isinstance(a, Point)
+        assert isinstance(b, Point)
+    except:
+        print("Given arguments aren't points")
+        exit()
+
+    return math.sqrt(pow(b.x - a.x, 2) + pow(b.y - a.y, 2))
```

Comment:
You can do it in a faster way, without using any modules, like:
return ((b.x - a.x) ** 2 + (b.y - a.y) ** 2) ** 0.5
But your solution is a good option too!